velox/functions/prestosql/ArrayFunctions.h

```diff
@@ -816,7 +816,7 @@ struct ArrayNGramsFunctionFunctionString {
   void call(
       out_type<Array<Array<Varchar>>>& out,
       const arg_type<Array<Varchar>>& input,
-      int64_t n) {
+      int32_t n) {
```

Comment:
nit: was the name of this class a typo or should it be ArrayNGramsFunctionString?

Reply:
Yea definitely type